### ISSUE TITLE
[CUDA] JAR invalid valid Java 9 module

### ISSUE THREAD

**Issue by Brownshome** [bytedeco/javacpp-presets#879]
When trying to load the [cuda jar](https://github.com/bytedeco/javacpp-presets/tree/master/cuda) the Java module system says the module is invalid.

The Jar's module file specifies that it exports `org.bytedeco.cuda.npps` but that package does not exist in the jar, causing this error:
```
> java --describe-module cuda -p .\cuda-10.2-7.6-1.5.2.jar
Caused by: java.lang.module.InvalidModuleDescriptorException: Package org.bytedeco.cuda.npps not found in module
```

I'm happy to make a PR and try and fix this myself if needed.

|  | Version |
| --- | --- |
| Jar | [10.2-7.6-1.5.2](https://mvnrepository.com/artifact/org.bytedeco/cuda/10.2-7.6-1.5.2) |
| Java | `openjdk 14 2020-03-17` and `openjdk 9` |
| OS | Windows 10 |

**Comment by saudet** [comment]
Yes, please do! Thanks

**Comment by saudet** [comment]
Fix released with version 1.5.4. Thank you for the contribution!
